File: lessons/lesson.23/morelanguages/car.java
public class Car {
    String make;
    String model;
    int year;
    int speed;

    public Car(String make, String model, int year) {
        this.make = make;
        this.model = model;
        this.year = year;
        this.speed = 0;
    }

    public void accelerate(int increment) {
        speed += increment;
    }

    public void brake(int decrement) {
        speed -= decrement;
    }

    public static void main(String[] args) {
        Car myCar = new Car("Toyota", "Corolla", 2023);
        System.out.println("My car: " + myCar.year + " " + myCar.make + " " + myCar.model);

        myCar.accelerate(20);
        System.out.println("Current speed: " + myCar.speed + " mph");

        myCar.brake(5);
        System.out.println("Current speed after braking: " + myCar.speed + " mph");
    }
}

File: lessons/lesson.23/morelanguages/car.cpp
#include <iostream>
using namespace std;

class Car {
public:
    string make;
    string model;
    int year;
    int speed;

    Car(string make, string model, int year) {
        this->make = make;
        this->model = model;
        this->year = year;
        speed = 0;
    }

    void accelerate(int increment) {
        speed += increment;
    }

    void brake(int decrement) {
        speed -= decrement;
    }
};

int main() {
    Car myCar("Toyota", "Corolla", 2023);
    cout << "My car: " << myCar.year << " " << myCar.make << " " << myCar.model << endl;

    myCar.accelerate(20);
    cout << "Current speed: " << myCar.speed << " mph" << endl;

    myCar.brake(5);
    cout << "Current speed after braking: " << myCar.speed << " mph" << endl;

    return 0;
}
